ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/SessionProperties.java

```diff
@@ -36,6 +37,27 @@
   private final boolean internalRequest;
   private final Map<String, String> sessionVariables;
 
+  /**
+   * @param mutableScopedProperties   The streamsProperties of the incoming request
+   * @param ksqlHostInfo              The ksqlHostInfo of the server that handles the request
+   * @param localUrl                  The url of the server that handles the request
+   * @param internalRequest           Flag indicating if request is from within the KSQL cluster
+   * @param sessionVariables          Initial session variables
+   */
+  public SessionProperties(
+      final Map<String, Object> mutableScopedProperties,
+      final KsqlHostInfo ksqlHostInfo,
+      final URL localUrl,
+      final boolean internalRequest,
+      final Map<String, Object> sessionVariables
+  ) {
+    this(mutableScopedProperties, ksqlHostInfo, localUrl, internalRequest);
```

Comment:
nit: can we have the other constructor call this one instead? I think constructor chains are more intuitive / easier to read when the ones with fewer arguments call the ones with more arguments (with intuitive defaults) since then you can just read the code in the final constructor in the chain in order to understand what's happening.